jbpm-test-coverage/src/main/java/org/jbpm/test/jobexec/CheckCallCommand.java

```diff
@@ -0,0 +1,40 @@
+package org.jbpm.test.jobexec;
+
+import org.kie.api.executor.Command;
+import org.kie.api.executor.CommandContext;
+import org.kie.api.executor.ExecutionResults;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class CheckCallCommand implements Command {
+    
+    private static final Logger logger = LoggerFactory.getLogger(CheckCallCommand.class);
+
+    private static Boolean commandExecuted = false;
+    private static Object LOCK = new Object();
```

Comment:
Instead of using both the Boolean and Object/LOCK, it might be more efficient to use a CyclicBarrier or CountDownLatch (a CyclicBarrier is essentially a reusable CountDownLatch).